python/plugins/db_manager/db_plugins/info_model.py

```diff
@@ -33,7 +33,14 @@ def __init__(self, db):
 
     def __del__(self):
         self.db = None
-
+        
+    def databaseInfo(self):
+        info = self.db.connector.getDatabaseInfo()
+        tbl = [
+            (QApplication.translate("DBManagerPlugin", "Database description: "), info[0])
```

Comment:
```suggestion
            (QApplication.translate("DBManagerPlugin", "Database description:" + " "), info[0])
```

I think it's low probability that translators see this trailing whitespace when they translate

Reply:
In that case it should be 
```python
(QApplication.translate("DBManagerPlugin", "Database description:") + " ", info[0])
``` 
?